src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -156,3 +156,16 @@ def mass_transfer_term(b, t, s, k, l, m):
             self.mscontactor.stream_component_interactions,
             rule=mass_transfer_term,
         )
+
+        self.clean_sx_pe_tank_cap = Var(initialize = 1405, units=units.gal, doc="Volume of polyethylene tanks")
+        self.num_sx_pe_tank = Var(initialize = 5, doc="Number of polyethylene tanks")
```

Comment:
A minor comment, but I would not say "polyethylene" here - these tanks could be made of anything really.

Reply:
Okay, actually it was written this way in the phase 1 report actually that is why, then how shall I mention them only as tanks?

Reply:
Just call them tanks. In the UKy process they happened to be made of polyethylene, but this model is general.

Reply:
Okay